src/gpodder/config.py

```diff
@@ -147,6 +147,8 @@
             'download_list': {
                 'remove_finished': True,
             },
+
+            'html_shownotes': True # enable webkit renderer
```

Comment:
Nitpick: Trailing comma after `True` to be in line with the rest of the config (adding items will make the diff nicer). Also, PEP-8 [requires two spaces before an inline comment](https://www.python.org/dev/peps/pep-0008/#inline-comments).